### DISCUSSION TITLE
2.5.0 doesn't work anymore ether.

### DISCUSSION THREAD

**Discussion by IamTheCarl** [timmo001/system-bridge#878] [Help] [answered]
While 2.5.0 runs, it's not collecting data and not reporting to Home Assistant. This could be a side effect of [installing and then removing 2.7](https://github.com/timmo001/system-bridge/issues/865).
Reading the log file I keep getting a warning message of "undefined" repeatedly.

I deleted .local/share/system-bridge and reconfigured the application to see if something in the configuration had been corrupted.
The "undefined" message went away but I still can't get it to collect data or report that data to Home Assistant.

**Comment by timmo001** [comment, marked as answer]
A fix has been released. Please update
